spec/models/user_spec.rb

```diff
@@ -644,48 +575,31 @@
     end
   end
 
-  describe '#password_updated_at' do
-    let(:user) { users(:affiliate_admin) }
-
-    it 'is set when the user is created' do
-      expect(user.password_updated_at).to_not be_nil
-    end
-
-    it 'is set when the password is updated' do
-      expect { user.update(password: "new1234!") }.
-        to change{ user.password_updated_at }
-    end
-
-    it 'is not set when other attributes are updated' do
-      expect { user.update(contact_name: 'Kermit the Frog') }.
-        to_not change{ user.password_updated_at }
-    end
-
-    it 'is not set when the password is blank' do
-      expect { user.update(email: 'new@example.com', password: '') }.
-        to_not change{ user.password_updated_at }
-    end
-  end
-
-  describe '#requires_password_reset?' do
-    subject { user.requires_password_reset? }
-
-    context 'when the password has never been reset' do
-      let(:user) { User.new }
+  describe '.from_omniauth' do
+    mock_user_auth('12345', 'test@gsa.gov')
```

Comment:
This auth object is not being used. If you want to use this auth object, you'd need to use it in the `let` block:
```ruby
let(:auth) { mock_user_auth('12345', 'test@gsa.gov') }
```